packages/liveblocks-core/src/index.ts

```diff
@@ -14,6 +14,10 @@
 export type { Client } from "./client";
 export { createClient } from "./client";
 export type { BaseAuthResult, Delegates } from "./connection";
+export type {
+  LegacyConnectionStatus as ConnectionStatus,
```

Comment:
While renamed internally everywhere, we keep the existing public type around as-is.